### PR TITLE
Issue #15340: created InputFormatted files for section 3.3.3 Ordering And Spacing

### PR DESCRIPTION
**Pull request by Zopsss**
#15340

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import javax.swing.*; // violation 'Using the '.*' form of import should be avoided'
+
+/** Some javadoc. */
+public class InputFormattedOrderingAndSpacing1 {
+  /** some javadoc. */
+  public static void main(String[] args) {
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java:18]: Please do us a favor to update original files
Like https://github.com/checkstyle/checkstyle/blob/master/src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputOrderingAndSpacing1.java

To have code in class.
To let us and other in future see that difference between files a minimum

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java:18]: I have formatted all the original files as well: 

https://github.com/checkstyle/checkstyle/pull/15557/files#diff-0df76b63da4b6c930800c10cee1884f054dd7f8e7d1d2ea0e6da3610319ee896

https://github.com/checkstyle/checkstyle/pull/15557/files#diff-a28667b832aebc07211c388bc72140492fc0fa8ab6f7e93b57e4cb4d75c82f1c

https://github.com/checkstyle/checkstyle/pull/15557/files#diff-a5337bee4a714ff79e55314968a2c19e2c9f04380defad6024a84a9a50f49759

and so on

**Comment by Zopsss** [comment]
circle ci failure can be ignored

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java (hunk 1 above)
Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java:18]: please make no diff in bodies of classes:
image

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java:18]: comment is different,
block of code is missed.

**Review by romani — APPROVED** [review]
@Zopsss , please fix minor item above
and I am good to merge.

File: src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java:18]: thanks for pointing this out, these were some extra instances which were unnecessary, I have removed them now.

**Comment by Zopsss** [comment]
> @Zopsss , please fix minor item above and I am good to merge. I checked all other, looks good.

done

**Comment by Zopsss** [comment]
hmm not sure why CI is failing: https://github.com/checkstyle/checkstyle/actions/runs/10601017909/job/29379876093?pr=15557

**Review by rdiachenko — APPROVED** [review]
lgtm

**Comment by rdiachenko** [comment]
I restarted `Check no closed issue references / check_issues (pull_request)`

**Comment by rdiachenko** [comment]
```
Following lines are referencing closed issues.
Such lines should be removed or the issue should be examined:
https://github.com/Zopsss/checkstyle/blob/15340-orderingspacing/config/spotbugs-exclude.xml#L239
Error: Process completed with exit code 1.
```

The referenced issue is not related to this PR's changes.

**Pull request merged**